modulefiles/wflow_orion.lua

```diff
@@ -8,8 +8,8 @@ whatis([===[Loads libraries needed for running SRW on Orion ]===])
 load("contrib")
 load("rocoto")
 
```

Comment:
needs to have 
unload("python")
to allow miniconda3 loading (similar to ./modulefiles/tasks/orion/miniconda_regiona_workflow)

Reply:
I did not end up putting a python module into the Orion module files, so there is no python to unload.